charts/merlin/tests/merlin_deployment_test.yaml

```diff
@@ -133,7 +133,7 @@ tests:
           path: metadata.name
           pattern: my-release-merlin$
       - equal:
-          path: spec.template.spec.containers[0].env[53].value
```

Comment:
to avoid this we can add new env variables to the end of the list

Reply:
Fixed